src/main.rs

```diff
@@ -1066,7 +1066,14 @@ async fn make_app(current_dir: &std::ffi::OsString) -> Result<Command> {
                 .short('t')
                 .long("template")
                 .help("Template to create")
-                .value_parser(["blank", "chat", "echo", "fibonacci", "file-transfer"])
+                .value_parser([
+                    "blank", 
+                    "chat", 
+                    "echo", 
+                    "fibonacci", 
+                    "file-transfer", 
+                    "framework-process"
```

Comment:
Let's change name here and throughout to `hyperapp-<something>`. As mentioned on #320, we're going to adopt a convention that hyperapp templates are prefixed by `hyperapp-`.